### ISSUE TITLE
Show improvement achieved by Using Umple through data

### ISSUE THREAD

**Issue by vahdat-ab** [umple/umple#1098]
Umple generated a lot useful APIs and even some elements required to be developed manually by developers of a specific programming language.
It would be great to let developers know how much they saved regarding these factors when they use Umple.
To start, we could just print how many LoC (Lines of Code) has been saved. This can somehow be converted to the time (if we can find an acceptable formula).

In other words, we can show how many lines of code would be needed to write the same code in a Umple.

**Comment by TimLethbridge** [comment]
Great idea. Simply output the number of LOC generated, and the number of LOC in the original and give the percent savings. The question is ... where in the UI to present this.

**Comment by TimLethbridge** [comment]
Where it says 'Done' in Umple online, after generating a programming language, output 'Done. ggg lines generated, from uuu Umple lines. (pp%)'. Where ppp is the uuu*100/ggg If this makes too much space, perhaps make the number of lines a popup and just say 'Done ppp%'
